controllers/rabbitmq/transporturl_controller.go

```diff
@@ -265,15 +252,43 @@ func (r *TransportURLReconciler) reconcileNormal(ctx context.Context, instance *
 		return ctrl.Result{}, err
 	}
 
+	// Get RabbitMq CR to check for service hostnames in status
+	rabbitmqCR := &rabbitmqv1.RabbitMq{}
+	err = r.Get(ctx, types.NamespacedName{Name: instance.Spec.RabbitmqClusterName, Namespace: instance.Namespace}, rabbitmqCR)
```

Comment:
do we catch bellow if `err !=nil` ? iiuc we only return an err if the host is not in the rabbitSecret. maybe we just check before the bellow conditions `if err != nil` ?